youtube_dl/YoutubeDL.py

```diff
@@ -848,6 +850,22 @@ def iter_playlistitems(format):
                 random.shuffle(entries)
 
             for i, entry in enumerate(entries, 1):
+                if self.params.get('playlistprompt', False):
+                    if entry['title']:
+                        skipnextvid = False
+                        while True:
+                            answer = compat_input('Do you want to download video %s/%s: "%s"? (y/n) ' %
+                                                  (i, n_entries, entry['title'].encode('ascii', 'ignore')))
```

Comment:
> Also trying to copy&Paste this specific Unicode Char ends in two question marks.

Some terminals (at least Konsole and QTerminal) can't handle non-BMP (i.e., Unicode characters > 0xFFFF). You may be another victim